exporters/requirements.txt

```diff
@@ -14,7 +14,7 @@ requests               # module requests
 
 
 # Failure exporter
-jira                   # module jira
+jira == 3.4.1          # module jira
```

Comment:
Please add comment to this file to point to our issue number telling why it was pinned.